mirgecom/eos.py

```diff
@@ -398,9 +396,13 @@ def pressure(self, cv: ConservedVars):
 
             p = (\gamma_{\mathtt{mix}} - 1)e
         """
-        temperature = self.temperature(cv)
-        y = self.species_fractions(cv)
-        return self._pyrometheus_mech.get_pressure(cv.mass, temperature, y)
+        @memoize_in(cv, (PyrometheusMixture.pressure,
+                         type(self._pyrometheus_mech)))
+        def get():
```

Comment:
This is _purely_ personal opinion. But I think it looks nicer if we name all the getters something like `get_pressure()`, `get_temperature()`, etc

Reply:
Please see how this grabs ya (4005f6f)

Reply:
Great!

Reply:
Well, your suggestion broke Y1 compatibility. THere, i've said it.